e _posts/0000-01-02-i-ssu.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: “Welcome to our second slide!”
+---
```

Comment:
THE TEXT ABOVE WAS PASTED IN THE EDITOR BY A SUPERWOMAN heeheee!
THE TEXT BELOW WAS ALSO PASTED IN THE EDITOR BY A SUPERWOMAN HEEHEEE!